pkg/controller/controller_instance.go

```diff
@@ -687,8 +738,8 @@ func (c *controller) reconcileServiceInstance(instance *v1alpha1.ServiceInstance
 				toUpdate,
 				v1alpha1.ServiceInstanceConditionReady,
 				v1alpha1.ConditionFalse,
-				errorProvisionCallFailedReason,
-				"ClusterServiceBroker returned a failure for provision call; operation will not be retried: "+s)
+				reason,
+				fmt.Sprintf("ClusterServiceBroker returned a failure for %v call; operation will not be retried: %v", provisionOrUpdateText, s))
 
 			if shouldStartOrphanMitigation(httpErr.StatusCode) {
```

Comment:
You should only do orphan mitigation here if you're provisioning, not if you're updating.